crates/nargo_cli/src/cli/new_cmd.rs

```diff
@@ -12,6 +16,9 @@ pub(crate) struct NewCommand {
     package_name: String,
     /// The path to save the new project
     path: Option<PathBuf>,
+
+    #[clap(flatten)]
+    init_config: InitCommand,
```

Comment:
We're going to end up with mix and  match pretty quick, can we pull out to one top file where args are defined?

Reply:
Instead we would have:
cli
|- arguments.rs (from where those structs are imported)
|- new_command.rs importing from arguments.rs